addons/project/models/project.py

```diff
@@ -686,10 +686,7 @@ def action_view_tasks_analysis(self):
 
     @api.model
     def _action_open_all_projects(self):
-        action = self.env['ir.actions.act_window']._for_xml_id(
-            'project.open_view_project_all' if not self.user_has_groups('project.group_project_stages') else
-            'project.open_view_project_all_group_stage')
-        return action
+        return self.env['ir.actions.act_window']._for_xml_id('project.open_view_project_all')
```

Comment:
I am not sure this is still useful to have this def if it is only to return the action from the xml_id and as it is only used once in the code.
We could use `self.env['ir.actions.act_window']._for_xml_id('project.open_view_project_all')` directly in `confirm_delete` of `project.delete.wizard`